mainwindow.cc

```diff
@@ -584,16 +584,19 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
 
   connect( ui.menuHistory, SIGNAL( aboutToShow() ),
            this, SLOT( updateHistoryMenu() ) );
-
+#if !defined( HAVE_X11 ) || QT_VERSION < QT_VERSION_CHECK( 5, 0, 0 )
   // Show tray icon early so the user would be happy. It won't be functional
   // though until the program inits fully.
+  // Do not create dummy tray icon in X. Cause QT5 failed to upgrade systemtray context menu.
+  // And as result menu for some DEs apppear to be empty, for example mate.
```

Comment:
*for example in MATE.*

Using the proper capitalization is important to reduce the chance of misunderstanding ("mate" is a common English word after all). See https://mate-desktop.org.

Reply:
Done